packages/patrol_cli/test/ios/ios_test_backend_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:patrol_cli/src/ios/ios_test_backend.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('IOSTestBackend', () {
+    group('BuildMode', () {
+      test('infers build options in debug mode when flavor is null', () {
+        const buildMode = BuildMode.debug;
+        String? flavor;
+
+        expect(buildMode.createConfiguration(flavor), 'Debug');
+        expect(buildMode.createScheme(flavor), 'Runner');
+      });
+
+      test('infers build options in debug mode when flavor is not null', () {
+        const buildMode = BuildMode.debug;
+        const flavor = 'dev';
+
+        expect(buildMode.createConfiguration(flavor), 'Debug-dev');
+        expect(buildMode.createScheme(flavor), 'dev');
+      });
+
+      test('infers build options in release mode when flavor is null', () {
+        const buildMode = BuildMode.release;
+        String? flavor;
```

Comment:
I'd assign `null` to it explicitly and make it a const and disable the lint for default values for this file or for those lines

Reply:
Nice but I don't agree. 

> Special cases aren't special enough to break the rules.
>
> ~ The Zen of Python

Reply:
I don't agree with your not agreeing